scripts/docuCheck.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+set -e # exit on errors
+
+BASE_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+
+declare -a LUIGI_FOLDERS=(
+  "client"
+  "core"
+)
+
+source $BASE_DIR/shared/bashHelpers.sh
+
+installPrerequisites() {
+  echoe "Install documentation prerequisites"
+  mkdir -p $BASE_DIR/tmp-docu
+
+  cd $BASE_DIR/tmp-docu
+  DOCU_VERSION=`cat $BASE_DIR/../client/package.json | jq --raw-output ".devDependencies.documentation"`
+  # create local package json to make it cachable
+  echo "{\"dependencies\": {\"documentation\": \"$DOCU_VERSION\"}}" > package.json
+  npm i
+
+  # link documentation binary if it does not exist
+  NODE_BIN_FOLDER="node_modules/.bin/"
+  for FOLDER in "${LUIGI_FOLDERS[@]}"
```

Comment:
Here there is a slash missing in _$FOLDER$NODE_BIN_FOLDER_ which creates additional folders and prevents caching for working properly. I would additionally propose to remove some duplication in the paths usage. Here is my proposal (you can replace it with the current implementation and see the actual changes ;) ):
``` 
  # link documentation binary if it does not exist
  for FOLDER in "${LUIGI_FOLDERS[@]}"
  do
    local NODE_BIN_FOLDER=$BASE_DIR/../$FOLDER/node_modules/.bin/
    if [ ! -e $NODE_BIN_FOLDER/documentation ]; then
      echo "Linking documentation binary for $FOLDER"
      mkdir -p $NODE_BIN_FOLDER
      ln -s $BASE_DIR/tmp-docu/$NODE_BIN_FOLDER/documentation $NODE_BIN_FOLDER/documentation
    fi
  done
```